src/include/duckdb/common/column_index.hpp

```diff
@@ -67,11 +138,56 @@ struct ColumnIndex {
 	bool IsVirtualColumn() const {
 		return index >= VIRTUAL_COLUMN_START;
 	}
+	void VerifySinglePath() const {
+		if (child_indexes.empty()) {
+			return;
+		}
+		if (child_indexes.size() != 1) {
+			throw InternalException(
+			    "We were expecting to find a single path in the index, meaning 0 or 1 children, found: %d",
+			    child_indexes.size());
+		}
+		child_indexes[0].VerifySinglePath();
+	}
+	bool IsChildPathOf(const ColumnIndex &path) const {
```

Comment:
remove, unused

Reply:
same as above: hold off on this, we'll need to use this instead of `vector<idx_t>` for VARIANT